src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +136,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
 
         ObservableList<String> usersList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
```

Comment:
I agree, I think the only change that happened here though was a space being removed. I wasn't really working on this part.